File: Compective online coding/code chef/Data_Sturechers/BROKENSTRING.py
t=int(input("Enter the number of test cases:"))
for i in range(t):
    lgt=int(input("Enter the length of the string:"))
    name=input("Enter the string:")
    fir=""
    sec=""
    for j in range(int(lgt/2)):
        fir=fir+name[j]
    for j in range(int(lgt/2),lgt):
        sec=sec+name[j]
    if fir==sec.lower() :
        print("YES")
    else :
        print("NO")

File: Compective online coding/code chef/Data_Sturechers/BROKENSTRING.c
#include <stdio.h>
#include <string.h>
int check(char name[],char num)
{
    int i,che;
    che=num/2;
	    char fir[che],last[che];
        fir[che]="";
        last[che]="";
        strcat(fir,name[i]);
        printf("%s",fir);
}
int main(void) {
	int test,i;
	printf("Enter the number of time we test");
	scanf("%d",&test);
	for(i=0;i<test;i++)
	{
	    int num;
	    scanf("%d",&num);
	    char name[num];
	    scanf("%s",name);
	    check(name,num);
	}
	return 0;
}